pkcs1-reader-bouncy-castle/src/test/java/com/palantir/remoting/ssl/pkcs1/BouncyCastlePkcs1ReadersTests.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.remoting.ssl.pkcs1;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.io.IOException;
+import java.security.spec.RSAPrivateKeySpec;
+import org.junit.Test;
+
+public final class BouncyCastlePkcs1ReadersTests {
```

Comment:
It's meant to check that the package properly registers the provider. I'll re-write the test to center around this assertion to make it clearer.